README.md

```diff
@@ -1,84 +1,75 @@
 
+## About
 [Group Income](https://groupincome.org/) is a fair income sharing mechanism that allows groups to provide their members a minimum income.
 
 This semi-centralized "Simple Edition" of Group Income is our way to rapidly prototype, develop, and research the concept with real groups.
 
-## Installation
+## Getting Started
 
 1. Install [Node.js](https://nodejs.org) (version 8 or greater).
 2. Install [Grunt](https://github.com/gruntjs/grunt): `npm install -g grunt-cli`
 3. Clone this repo (or a fork of it if you plan on [contributing](#contributing)) and `cd` into it.
 4. Install dependencies: `npm install`
+5. Now try out the [dev workflow](#basic-workflow).
+- 👩‍🎨 Check out how Group Income will feel like by taking a look at the [design files on Figma](https://www.figma.com/file/mxGadAHfkWH6qApebQvcdN/Group-Income-2.0?node-id=1204%3A0)
 
-Now try out the [dev workflow](#basic-workflow).
-
----
+## Contributing
 
 We use [standard](https://github.com/feross/standard) for the code style and [Github project boards](https://help.github.com/articles/about-project-boards/) for efficient project management.
 
 [![js-standard-style](https://cdn.rawgit.com/feross/standard/master/badge.svg)](https://github.com/feross/standard)
 
-## Contributing
+We are committed to ensuring digital accessibility for all people. We are continually improving the user experience for everyone, and applying the relevant accessibility standards. We will be conformant with [WCAG 2.0](https://www.w3.org/WAI/standards-guidelines/wcag/) Level AA and trying or best to reach Level AAA in some areas of the application.
+
+<a href="https://www.w3.org/WAI/WCAG2AA-Conformance"
+  title="Explanation of WCAG 2.0 Level Double-A Conformance">
+  <img height="32" width="88"
+    src="https://www.w3.org/WAI/wcag2AA"
+    alt="Level Double-A conformance, W3C WAI Web Content Accessibility Guidelines 2.0">
+</a>
+
 
-#### Read first!
+#### Read first
 
-- **[:book: CONTRIBUTING.md](CONTRIBUTING.md) (required reading to send a PR!)**
+- [:book: CONTRIBUTING.md](CONTRIBUTING.md) (required reading to send a PR!)
 - [:book: Getting Started — Modern frontend concepts & project overview](docs/Getting-Started-frontend.md)
+- [:book: Style Guide - Our development guidelines](docs/Style-Guide.md)
+
 
 #### Basic workflow
+Group Income is an application based on [Vue.js](https://vuejs.org/). Here's how you can get started the development:
```

Comment:
Again, needs a blank line after the heading.

Reply:
Reviewed all missing empty lines!